packages/commerce-sdk-react/src/hooks/ShopperBaskets/cache.ts

```diff
@@ -51,9 +51,13 @@ const updateCustomerBasketsQuery = (
     return {
         queryKey: getCustomerBaskets.queryKey({...parameters, customerId}),
         updater: (oldData: CustomerBasketsResult | undefined) => {
-            // do not update if response basket is not part of existing customer baskets
+            // For createBasket, response basket is not part of existing customer baskets
+            // so we create a new result with 1 basket
             if (!oldData?.baskets?.some((basket) => basket.basketId === parameters.basketId)) {
```

Comment:
nit, If we assume that cache is always empty when createBasket is called then this might be a better reflection (?)

```suggestion
            if (!oldData?.baskets? || !oldData?.baskets.length)) {
```

Reply:
Two typos in your suggestion I think @kevinxh 

```suggestion
            if (!oldData?.baskets || !oldData?.baskets?.length)) {
```

Reply:
```suggestion
            if (!oldData?.baskets?.length) {
```
Avoids doing the same check twice. 😉

Reply:
I've applied the suggested change from @wjhsf .

I'll leave the conditional in the updater since, with the way it is now, I think it makes sense to update the cache with 1 basket if there were no baskets in there prior regardless of whether the call was createBasket or something else.